test/Makefile.am

```diff
@@ -58,6 +58,8 @@ t8code_googletest_programs = \
   test/t8_forest/t8_gtest_ghost_exchange \
   test/t8_forest/t8_gtest_ghost_delete \
   test/t8_forest/t8_gtest_ghost_and_owner \
+  test/t8_forest/t8_gtest_forest_commit \
```

Comment:
The test was already implemented. However, it was not added to the makefile beforehand. I just made the addition to the makefile, this could have been a PR on it's own, but it is somehow linked in terms of the test coverage of the balance functionality.